src/fdc/MSXtar.cc

```diff
@@ -107,10 +117,18 @@ void MSXtar::parseBootSector(const MSXBootSector& boot)
 	chrootSector = rootDirStart;
 	rootDirLast = rootDirStart + nbRootDirSectors - 1;
 
+	// Whether to use FAT16 is strangely implicit, it must be derived from the
+	// cluster count being higher than 0xFF4. Let's round that up to 0x1000,
+	// since FAT12 uses 1.5 bytes per cluster its maximum size is 12 sectors,
+	// whereas FAT16 uses 2 bytes per cluster so its minimum size is 16 sectors.
+	// Therefore we can pick the correct FAT type with sectorsPerFat >= 14.
+	unsigned fat16Threshold = (0x1000 * 3 / 2 + 0x1000 * 2) / 2 / SECTOR_SIZE;
```

Comment:
Very much a detail: this variable could be `constexpr`.

Reply:
Incidentally I did see this one (normally I probably wouldn’t think about it too readily), however I figured the compiler would do it for me anyway so the line would be shorter without.

Reply:
You're right.
It's not useful or the compiler but for human readers of the code.